packages/valory/skills/hello_world_abci/behaviours.py

```diff
@@ -49,13 +51,19 @@
 class HelloWorldABCIBaseBehaviour(BaseBehaviour, ABC):
     """Base behaviour behaviour for the Hello World abci skill."""
 
+    # TODO: Question: What's the difference between synchronized data and shared state?
```

Comment:
Synchronized data is the joint "synchronized memory" of the service that all agents agree on as the service business logic moves forward.

Shared state is just local to an agent. Here the term "shared state" means the state shared among all components of the agent (skills, protocols, etc...).